shared/src/main/java/gov/hhs/cdc/trustedintermediary/utils/RetryFailedException.java

```diff
@@ -0,0 +1,11 @@
+package gov.hhs.cdc.trustedintermediary.utils;
+
+public class RetryFailedException extends Exception {
+    public RetryFailedException(String message) {
+        super(message);
+    }
```

Comment:
Looks like this constructor is only used in tests.  We could probably remove it and then the test can pass in a bogus `new Exception()` as the second argument.

Reply:
Done